inst/Upgrades.qml

```diff
@@ -0,0 +1,289 @@
+
+		ChangeRename{	from: "counts";													to: "count"								}
+		ChangeRename{	from: "regressionCoefficientsConfidenceIntervals";				to: "regressionCoefficientsCi"			}
+		ChangeRename{	from: "regressionCoefficientsConfidenceIntervalsInterval";		to: "regressionCoefficientsCiLevel"		}
+		ChangeRename{	from: "VovkSellkeMPR";											to: "vovkSellke"						}
+	}
+
+	Upgrade
+	{
+		functionName:		"RegressionLogLinearBayesian"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "counts";															to: "count"										}
+		ChangeRename{	from: "maxModels";														to: "modelCutOffBestDisplayed"					}
+		ChangeRename{	from: "posteriorProbabilityCutOff";										to: "modelCutOffPosteriorProbability"			}
+		ChangeRename{	from: "regressionCoefficientsCredibleIntervals";						to: "regressionCoefficientsCi"					}
+		ChangeRename{	from: "regressionCoefficientsCredibleIntervalsInterval";				to: "regressionCoefficientsCiLevel"				}
+		ChangeRename{	from: "regressionCoefficientsSubmodelCredibleIntervals";				to: "regressionCoefficientsSubmodelCi"			}
+		ChangeRename{	from: "regressionCoefficientsSubmodelCredibleIntervalsInterval";		to: "regressionCoefficientsSubmodelCiLevel"		}
+		ChangeRename{	from: "sampleMode";														to: "samplingMethod"							}
+		ChangeRename{	from: "fixedSamplesNumber";												to: "samplingMethodManualSamples"				}
+	}
+
+	Upgrade
+	{
+		functionName:		"ABTestBayesian"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "normal_mu";					to: "normalPriorMean"				}
+		ChangeRename{	from: "normal_sigma";				to: "normalPriorSd"					}
+		ChangeRename{	from: "descriptives";				to: "descriptiveTable"				}
+		ChangeRename{	from: "plotPriorAndPosterior";		to: "priorAndPosteriorPlot"			}
+		ChangeRename{	from: "plotPosteriorType";			to: "priorAndPosteriorPlotType"		}
+		ChangeJS
+		{
+			name:		"priorAndPosteriorPlotType"
+			jsFunction:	function(options)
+			{
+				switch(options["priorAndPosteriorPlotType"])
+				{
+					case "LogOddsRatio":		return "logOddsRatio";
+					case "OddsRatio":			return "oddsRatio";
+					case "RelativeRisk":		return "relativeRisk";
+					case "AbsoluteRisk":		return "absoluteRisk";
+					case "p1&p2":				return "p1p2";
+				}
+			}
+		}
+		ChangeRename{	from: "plotSequentialAnalysis";		to: "sequentialAnalysisPlot"		}
+		ChangeRename{	from: "plotPriorOnly";				to: "priorPlot"						}
+		ChangeRename{	from: "plotPriorType";				to: "priorPlotType"					}
+		ChangeJS
+		{
+			name:		"priorPlotType"
+			jsFunction:	function(options)
+			{
+				switch(options["priorPlotType"])
+				{
+					case "LogOddsRatio":		return "logOddsRatio";
+					case "OddsRatio":			return "oddsRatio";
+					case "RelativeRisk":		return "relativeRisk";
+					case "AbsoluteRisk":		return "absoluteRisk";
+					case "p1&p2":				return "p1p2";
+					default:					return options["priorPlotType"];
+				}
+			}
+		}
+		ChangeRename{	from: "plotRobustness";						to: "robustnessPlot"					}
```

Comment:
Wouldn't "bfRobustnessPlot" be more descriptive? This would make it consistent with the Bayesian correlation.